validator_client/src/cli.rs

```diff
@@ -127,6 +127,21 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .takes_value(true)
                 .conflicts_with("graffiti")
         )
+        .arg(
+            Arg::with_name("suggested-fee-recipient")
+                .long("suggested-fee-recipient")
+                .help("Specify the address, that will receive transaction fees collected from any \
+                        blocks produced by this validator client.")
```

Comment:
```suggestion
                .help("The fallback address provided to the BN if nothing suitable is found \
                           in the validator definitions or fee recipient file.")
```

It's a messy diff sorry, but I'm trying to express the ordering.